docs/Ecosystem.md

```diff
@@ -27,6 +27,7 @@ Plugins maintained by the fastify team are listed under [Core](#core) while plug
 - [`fastify-react`](https://github.com/fastify/fastify-react) React server side rendering support for Fastify with [Next](https://github.com/zeit/next.js/).
 - [`fastify-redis`](https://github.com/fastify/fastify-redis) Fastify Redis connection plugin, with which you can share the same Redis connection across every part of your server.
 - [`fastify-reply-from`](https://github.com/fastify/fastify-reply-from) Plugin to forward the current http request to another server.
+- [`fastify-sensible`](https://github.com/fastify/fastify-sensible) Defaults for Fastify that everyone can agree on.
```

Comment:
I would be more concrete "Decorators for common use cases like HTTP errors or assertions"

Reply:
Generally I would agree with you, but the plugin name and description are a quote from https://github.com/tpope/vim-sensible ;)

Reply:
Yes, but that list does not specify what it does. Can you add another sentence?

Reply:
Done... :D